### PR TITLE
Only execute schema changes on MySQL, PHPCS stuff

### PR DESCRIPTION
**Pull request by mbabker**
.

### CODE REVIEW

File: src/com_weblinks/script.php [hunk 1]
```diff
@@ -70,16 +68,25 @@ public function install($parent)
...
 	 */
-	function postflight($type, $parent)
+	public function postflight($type, $parent)
 	{
-		// Drop the Table Colums if needed
-		$this->dropColumnsIfNeeded();
+		// Only execute database changes on MySQL databases
+		$dbName = JFactory::getDbo()->name;
+
+		if (strpos($dbName, 'mysql') !== false)
```
Comment (zero-24) [inline src/com_weblinks/script.php]: @mbabker does this also cover pdo? (Just to be sure)

File: src/com_weblinks/script.php [hunk 2]
```diff
@@ -70,16 +68,25 @@ public function install($parent)
...
+		{
+			// Drop the Table Colums if needed
+			$this->dropColumnsIfNeeded();
```
Comment (zero-24) [inline src/com_weblinks/script.php]: Works i guess if we get the PR by @wilsonge `addColumnsIfNeeded()` needs also go into the if?

**Comment by zero-24** [comment]
@test successful with the changes above and in combination with #41 Great work!

File: src/com_weblinks/script.php (hunk 1 above)
Comment (mbabker) [inline src/com_weblinks/script.php]: Yes, `strpos('pdomysql', 'mysql')` should return an integer and not a boolean false.

File: src/com_weblinks/script.php (hunk 2 above)
Comment (mbabker) [inline src/com_weblinks/script.php]: Correct as these functions are MySQL specific (the other databases didn't have these schema changes and the SQL that's in the methods is leaning toward MySQL specific syntax)

**Comment by zero-24** [comment]
Also successful tested by @infograf768 https://github.com/joomla-extensions/weblinks/issues/44#issuecomment-103416428

With my package and https://github.com/joomla-extensions/weblinks/pull/42#discussion_r30539810 included.

**Comment by javigomez** [comment]
Merging and awaiting #41 for update

**Pull request merged**